LavalinkServer/application.yml.example

```
@@ -23,6 +23,7 @@ lavalink:
       #excludedIps: ["...", "..."] # ips which should be explicit excluded from usage by lavalink
       #strategy: "RotateOnBan" # RotateOnBan | LoadBalance | NanoSwitch | RotatingNanoSwitch
       #searchTriggersFail: true # Whether a search 429 should trigger marking the ip as failing
+      #retryLimit: -1 # -1 = user default lavaplayer value | 0 = infinity | >0 = retry will happen this numbers times
```

Not sure if this is intended or not but this seems like a typo
```suggestion
      #retryLimit: -1 # -1 = use default lavaplayer value | 0 = infinity | >0 = retry will happen this numbers times
```